src/learn/ApproveExperienceModal.js

```diff
@@ -12,112 +19,138 @@ import { I18n } from "@aws-amplify/core";
+          <MdClose />
+        </IconButton>
+      </DialogTitle>
+      <DialogContent className={classes.root}>
+        <h3>{I18n.get("description")}</h3>
+        <BlockContent blocks={activeExperience.overview} />
+        <h3>{I18n.get("notesForCoach")}</h3>
+        <p>{mongoExperience[activeExperience.priority].athleteNotes}</p>
+        <h3>{I18n.get("submitLink")}</h3>
+        <a
+          href={mongoExperience[activeExperience.priority].github}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {I18n.get("openLink")}
+        </a>
+        <br />
+        {mongoExperience[activeExperience.priority].coachNotes.length > 0 ? (
+          <>
+            <h3>{I18n.get("coachesNotes")}</h3>
+            <p>{mongoExperience[activeExperience.priority].coachNotes}</p>
+            <Button
+              size="large"
+              variant="contained"
+              color="primary"
+              onClick={closeModal}
+            >
               {I18n.get("close")}
             </Button>
-
-            {props.mongoExperience[props.activeExperience.priority].approved ===
-            true ? null : (
-              <>
-                {props.mongoExperience[props.activeExperience.priority]
-                  .revisionsNeeded === false ? (
+          </>
+        ) : (
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <h3>{I18n.get("coachesNotes")}</h3>
+            <TextField
+              id="coachNotes"
+              variant="filled"
+              size="medium"
+              autoFocus
+              {...register("coachNotes")}
+            />
+            <DialogActions>
+              <Button
+                size="large"
+                variant="contained"
+                color="primary"
+                onClick={closeModal}
+              >
+                {I18n.get("close")}
+              </Button>
+              {mongoExperience[activeExperience.priority].approved ===
+              true ? null : (
+                <>
                   <Button
-                    className="btn"
-                    onClick={() => {
-                      props.markRequestRevisions(
-                        props.activeExperience,
-                        props.mongoExperience,
-                        coachNotes
-                      );
-                    }}
+                    size="large"
+                    variant="contained"
+                    color="primary"
+                    type="submit"
                   >
-                    {I18n.get("requestRevisions")}
+                    {mongoExperience[activeExperience.priority]
+                      .revisionsNeeded === false
+                      ? I18n.get("requestRevisions")
+                      : I18n.get("confirmAchievement")}
                   </Button>
-                ) : null}
-                <Button
-                  className="btn"
-                  onClick={() => {
-                    props.markComplete(
```

Comment:
@AleksandrKurilov can you please add this button back in? Right now, you only have the `Request Revisions` button enabled, and we also need to be able to 'Approve' or markComplete.

Reply:
Resolving, I've verified this has been added back in.